Doc/whatsnew/3.10.rst

```diff
@@ -1133,6 +1133,11 @@ Removed
   (Contributed by Yurii Karabas, Andrew Svetlov, Yury Selivanov and Kyle Stanley
   in :issue:`42392`.)
 
+* Remove the undocumented ``io.OpenWrapper`` and ``_pyio.OpenWrapper``
```

Comment:
`OpenWrapper` itself is not purposed to be used in user code. The main effect is that it changes builtin `open` if Python implementation is used. It can affect PyPy or other implementations.

Reply:
_pyio is supposed to be a drop-in replacement for the io module, whereas _pyio.open and io.open behave differently.

See https://bugs.python.org/issue43682 which would make it possible to use _pyio.open as a method, rather than having to go through OpenWrapper.